.ci/scripts/prepare-benchmarks-env.sh

```diff
@@ -14,17 +17,34 @@ if [[ -z "$NODE_VERSION" ]]; then
   exit 1
 fi
 
+echo "--- Download nvm"
 # This particular configuration is required to be installed in the baremetal
 curl -sS -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.39.3/install.sh | bash
 export NVM_DIR="$HOME/.nvm"
+
+echo "--- Debug nvm "
+# debug if curl works as explained in https://github.com/nvm-sh/nvm/issues/3117
+curl -I --compressed -v https://nodejs.org/dist/
+
+echo "--- Install nvm"
 set +x  # Disable xtrace because output using nvm.sh is huge.
-[ -s "$NVM_DIR/nvm.sh" ] && \. "$NVM_DIR/nvm.sh"
+if [ -s "$NVM_DIR/nvm.sh" ] ; then
+  # As long as the Buildkite agent does some weird behaviours compare to the Jenkins agent
+  # let's avoid failures when running nvm.sh for the first time. For some reason
+  # nvm_resolve_local_alias default returns VERSION= while in Jenkins a similar
+  # runner returns VERSION=v14.21.3
+  # Already reported to the System owners of the Buildkite agent setup.
+  \. "$NVM_DIR/nvm.sh" || true
```

Comment:
I've no clue what's the reason for the failure... I also asked to the community in https://github.com/nvm-sh/nvm/issues/3117

It seems to work later on if using `|| true`

Reply:
The node version used is the one present in `.nvmrc`.